OnlineLibrary.DAL/Repositories/DapperUnitOfWork.cs

```diff
@@ -0,0 +1,56 @@
+﻿using OnlineLibrary.DAL.Interfaces;
+using System;
+
+namespace OnlineLibrary.DAL.Repositories.Dapper
+{
+    public class DapperUnitOfWork : IUnitOfWork
+    {
+        private string _connectionString;
+
+        private Lazy<BookRepository> _bookRepository;
+
+        private Lazy<AuthorRepository> _authorRepository;
+
+        private Lazy<TagRepository> _tagRepository;
+
+        private Lazy<UserRepository> _userRepository;
+
+        private Lazy<ReservationRepository> _reservationRepository;
+
+        public DapperUnitOfWork(string connectionString)
```

Comment:
instead of raw string it's better to create class with ConnectionString property and inject this
for better understanding read https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/options?view=aspnetcore-6.0